src/util/misc.js

```diff
@@ -522,6 +522,35 @@
       imageData = null;
 
       return _isTransparent;
+    },
+
+    /**
+     * Parse preserveAspectRatio attribute from element
+     * @param {string} attribute to be parsed
+     * @return {Object} an object containing align and meetOrSlice attribute
+     */
+    parsePreserveARAttribute: function(attribute) {
```

Comment:
I would change AR to AspectRatio. Yes, it's a bit long-ish, but AR just looks way too cryptic IMO.